features/leaveroom/fake/src/main/kotlin/io/element/android/features/leaveroom/fake/LeaveRoomPresenterFakeModule.kt

```diff
@@ -1,30 +0,0 @@
-/*
- * Copyright (c) 2023 New Vector Ltd
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.element.android.features.leaveroom.fake
-
-import com.squareup.anvil.annotations.ContributesTo
-import dagger.Binds
-import dagger.Module
-import io.element.android.features.leaveroom.api.LeaveRoomPresenter
-import io.element.android.libraries.di.SessionScope
-
-@Module
-@ContributesTo(SessionScope::class)
-interface LeaveRoomPresenterFakeModule {
```

Comment:
Is this never used, nor potentially useful?

Reply:
Not used: `LeaveRoomPresenterFake` (now `FakeLeaveRoomPresenter`) - as all other `Fake` classes - is never injected, constructor is explicitly called.